example/test/person_test.exs

```diff
@@ -9,46 +9,33 @@ defmodule PersonTest do
     Repo.delete_all(Company)
     Repo.delete_all(PaperTrail.Version)
 
-    Company.changeset(%Company{}, %{
-      name: "Acme LLC",
-      website: "http://www.acme.com"
-    }) |> Repo.insert
+    %{name: "Acme LLC", website: "http://www.acme.com"}
+    |> new_company()
+    |> Repo.insert()
 
-    Company.changeset(%Company{}, %{
-      name: "Another Company Corp.",
-      is_active: true,
-      address: "Sesame street 100/3, 101010"
-    }) |> Repo.insert
+    %{name: "Another Company Corp.", is_active: true, address: "Sesame street 100/3, 101010"}
+    |> new_company()
+    |> Repo.insert()
 
     :ok
   end
 
-  test "creating a person with meta tag creates a person version with correct attributes" do
-    company = first(Company, :id) |> Repo.one
+  test "[multi tenant] creating a person with meta tag creates a person version with correct attributes" do
```

Comment:
We are infact never testing multi tenancy in this file as far as I see, (Im not suggesting that we should)